### DISCUSSION TITLE
Using THEMA for Divina publications

### DISCUSSION THREAD

**Discussion by HadrienGardeur** [readium/webpub-manifest#98] [General]
RWPM provides a generic way of indicating a subject but it doesn't recommend a good way of identifying them or which subject scheme should be used.

In the case of Divina, I think that among subject schemes used in the industry, there's a clear winner: THEMA.

While the hierarchy of THEMA tends to be a little too convoluted for its own good, it has the benefit of having a list that organizations from all over the world can contribute to with extensive support for comics/manga/BD already.

Let's take manga for example: https://ns.editeur.org/thema/en/XAM

If we agree on a common way of identifying THEMA (such as https://ns.editeur.org/thema/ for example) this would look like this in a Divina:

```json
"subject": [
  {
    "name": "Shonen",
    "scheme": "https://ns.editeur.org/thema/",
    "code": "XAMG" 
  }
]
```

I've exchanged a few emails with Chris at EDItEUR and he's very interested in getting more feedback from the community regarding ways that THEMA could be improved for comics/manga/BD as well, so we have a direct line available when/if needed.

**Comment by majora2007** [comment]
From the brief look, I do like what THEMA brings to the table. It's very well structured and would be great to cover both traditional books as well as comics/manga. However, one interesting issue is the name. 

Many manga sites would capture manga with Vampires as "Vampire", however for Thema, the name would be [Mythical creatures: Vampires, werewolves and other shapeshifters](https://ns.editeur.org/thema/en/VXQM2), which is quite verbose and makes rendering on devices very tedious. 

Does THEMA offer a shorthand or could we provide a shorthand name in the subject schema?

**Reply by HadrienGardeur** [reply]
THEMA doesn't provide a shorthand but most retailers using it tend to remap the terms to something more suitable for their audience.

For comic reading app and self-hosting solutions, this might mean reworking some of the labels listed under https://ns.editeur.org/thema/en/X

**Comment by HadrienGardeur** [comment]
To continue the discussion from our last Anansi/Readium call: I think that comics reading app and self-hosting solutions could limit themselves to every code under X in THEMA.

This doesn't even have to be as-is since the X branch in THEMA is divided into:

- styles/traditions
- and genres

To go back to @majora2007 on having a shorthand expression, this could be exposed to users as two separate fields:

**Styles/traditions**

| Code | Label |
| ----- | ------ |
| XAB | Graphic novels |
| XAD | European style (Bande dessinée) |
| XAK | American style (Comics) |
| XAK | Asian style (Manga and manhwa) |
| XAX | Other styles/traditions |

I'm not sure why THEMA doesn't have comic strips (XY) below the XA code, but it would make more sense to have it here IMO.
Among potential additions we mentioned:

- Webtoons
- Web comics

Under XAK, THEMA also provides specialized codes for manga:

| Code | Label |
| ----- | ------ |
| XAMC | Kodomo |
| XAMF | Shōjo |
| XAMG | Shōnen |
| XAML | Seinen |
| XAMR | Josei |
| XAMT | Yaoi |
| XAMV | Bara |
| XAMX | Adult (erotic, extreme violence) |
| XAMY | Yuri |

During the call, the group was surprised by the presence of Kodomo, which they never saw elsewhere before.

**Genres**

| Code | Label |
| ----- | ------ |
| XQA | Memoirs, true stories and non-fiction |
| XQB | Literary adaptations |
| XQC | Inspired by or adapted from other media |
| XQD | Crime, mystery and thrillers |
| XQE | Society / Culture |
| XQF | Modern and contemporary fiction / literature |
| XQG | Action and adventure |
| XQH | Horror / supernatural |
| XQJ | Sports |
| XQK | Superheroes and super-villains |
| XQL | Science fiction |
| XQM | Fantasy, esoteric |
| XQN | Anthropomorphic / animal stories |
| XQP | Traditional stories, myths, legends |
| XQR | Romance |
| XQS | School / college life |
| XQT | Humorous |
| XQV | Historical |
| XQW | Religious, spiritual, inspirational |
| XQX | Adult |

We didn't had time to discuss these genres during the call.
